fast/stages/1-vpc-sc/README.md

```diff
@@ -0,0 +1,308 @@
+# VPC Service Controls
+
+This stage sets up VPC Service Controls (VPC-SC) for the whole organization and is a thing FAST-compliant wrapper on the [VPC-SC module](../../../modules/vpc-sc/), with some minimal defaults.
+
+<p align="center">
+  <img src="diagram.png" alt="Security diagram">
```

Comment:
remove projects from diagram

Reply:
removed diagram as I found no way of doing a good representation of VPC-SC resources, we can of course revisit later